src/status_im/contexts/wallet/events.cljs

```diff
@@ -121,13 +121,20 @@
       (update :balances-per-chain update-vals #(update % :raw-balance money/bignumber))
       (update :balances-per-chain update-keys (comp utils.number/parse-int name))))
 
+(defn- remove-tokens-with-empty-values
+  [tokens]
+  (remove
+   #(or (string/blank? (:symbol %)) (string/blank? (:name %)))
+   tokens))
```

Comment:
What is the reason for status-go to return these tokens without names or symbols?

Reply:
It could be due to the market provider being down or it can be a custom token in the address which `status-go` is unable to fetch the data at that moment.

Reply:
But we get a balance, isn't it weird that just the name or symbols are missing?

the total balance calculation then won't be correct :thinking: is it expected  in this case? WDYT about adding a warning or something to it?

Reply:
Yes, It's weird that the balances are returned but not the name or symbol. The issue is that we deal with uncertain tokens. I added this token filter as the desktop app has a similar one to remove the tokens with no name or symbol.

IMO, the total balance calculation would be correct (atleast) for the tokens displayed (with all required values such as name, symbol,...etc).

> WDYT about adding a warning or something to it?

Yes, Ulises. That's my next step. Displaying a warning/error icon would improve the UX. This is worth discussing with the design team.